Library/Homebrew/cmd/info.rb

```diff
@@ -62,16 +62,18 @@ def github_fork
   end
 
   def github_info f
-    if f.tap?
-      user, repo = f.tap.split("/", 2)
```

Comment:
How can consuming code know or not know the implementation details? I still think avoiding duplicated computation is worthy to do.

However, since you are the maintainers. It's your call. I will revert it.